### ISSUE TITLE
Insert Send doesn't send any data anymore from bus to bus

### ISSUE THREAD

**Issue by suedwestlicht** [rncbc/qtractor#405]
For ages I use the following setup:

Track -> Aux Send -> Bus with echo plugin
Bus with echo plugin -> Insert Send -> Bus with Reverb plugin

The Bus with the Reverb plugin should be placed "left" (before) from the Bus with the echo plugin.

So I need only one Reverb plugin instance for tracks and its echo.

This doesn't work anymore.

I add a simple session that shows the routing. Bus2 should send audio to Bus1 via Insert Send but no audio is sent.
[Bustest.qtz.zip](https://github.com/rncbc/qtractor/files/12160582/Bustest.qtz.zip)

**Comment by rncbc** [comment]
last release that seemed to work was v0.9.33

**Comment by suedwestlicht** [comment]
Yes, it works again.
Thx Rui

**Comment by G3N-es** [comment]
I appreciate the repair, I consider this functionality essential: to be able to send signals from "buses" to the final master.

However there is a new error. Now if you create inserts between tracks the sound is distorted.

**Comment by suedwestlicht** [comment]
Indeed. That's a feature I never used but I attach a sample session. The distorted Insert Send  sounds as as if was pitched down.
[Bustest.qtz.zip](https://github.com/rncbc/qtractor/files/12273722/Bustest.qtz.zip)

**Comment by rncbc** [comment]
confirmed; and it was fine all the way until the later repair/fix re. inserts on buses... good catch.

back to repair shop indeed
thanks

**Comment by rncbc** [comment]
regression in https://github.com/rncbc/qtractor/commit/6cf2e0c; prolly fixing both accounts

please provide some precious feedback ;)
cheers

**Comment by G3N-es** [comment]
Works!!!   :D 

Thanks Rui

**Comment by suedwestlicht** [comment]
Yep, both cases are fixed. Thx Rui.

**Comment by rncbc** [comment]
closing now.
